aether-common-module/aether/common/conf/settings.py

```
@@ -141,8 +141,8 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': os.environ.get('DB_NAME'),
-        'PASSWORD': os.environ.get('PGPASSWORD', ''),
+        'NAME': os.environ['DB_NAME'],
+        'PASSWORD': os.environ['PGPASSWORD'],
         'USER': os.environ.get('PGUSER', 'postgres'),
         'HOST': os.environ.get('PGHOST', 'db'),
         'PORT': os.environ.get('PGPORT', '5432'),
```

How could we force this to fail in production if these variables are not set, while using sane defaults in dev? An `if DEBUG` condition, maybe?

---

What do you mean? Do you prefer to remove the default values?

---

If by some reason these variables don't get set in production we will see a connection error, which could be misleading. It would save us a lot of time to see the error that the variables are not set. I'm thinking:
```py
db_user = 'postgres' if DEBUG else os.environ['PGUSER']
# the same for host and port
```

---

I don't like my approach now that I look at it 😅 We want to set `DEBUG=True` in the development cluster, and take those values from the environment at the same time.